pkg/services/ngalert/metrics/multi_org_alertmanager.go

```diff
@@ -81,12 +97,66 @@ type AlertmanagerAggregatedMetrics struct {
 	silencesQueryDuration           *prometheus.Desc
 	silences                        *prometheus.Desc
 	silencesPropagatedMessagesTotal *prometheus.Desc
+
+	// exported metrics, gathered from Alertmanager Dispatcher
+	dispatchAggrGroups         *prometheus.Desc
+	dispatchProcessingDuration *prometheus.Desc
 }
 
 func NewAlertmanagerAggregatedMetrics(registries *metrics.TenantRegistries) *AlertmanagerAggregatedMetrics {
 	aggregatedMetrics := &AlertmanagerAggregatedMetrics{
 		registries: registries,
 
+		numNotifications: prometheus.NewDesc(
+			fmt.Sprintf("%s_%s_notifications_total", Namespace, Subsystem),
+			"The total number of attempted notifications.",
+			[]string{"org", "integration"}, nil),
+		numFailedNotifications: prometheus.NewDesc(
+			fmt.Sprintf("%s_%s_notifications_failed_total", Namespace, Subsystem),
+			"The total number of failed notifications.",
+			[]string{"org", "integration"}, nil),
+		numNotificationRequestsTotal: prometheus.NewDesc(
+			fmt.Sprintf("%s_%s_notification_requests_total", Namespace, Subsystem),
+			"The total number of attempted notification requests.",
+			[]string{"org", "integration"}, nil),
+		numNotificationRequestsFailedTotal: prometheus.NewDesc(
+			fmt.Sprintf("%s_%s_notification_requests_failed_total", Namespace, Subsystem),
+			"The total number of failed notification requests.",
+			[]string{"org", "integration"}, nil),
+		notificationLatencySeconds: prometheus.NewDesc(
+			fmt.Sprintf("%s_%s_notification_latency_seconds", Namespace, Subsystem),
+			"The latency of notifications in seconds.",
+			nil, nil),
```

Comment:
For reviewers: [Upstream notification metrics](https://github.com/prometheus/alertmanager/blob/main/notify/notify.go#L254)